### PR TITLE
update karmadactl-usage-conventions.md

### PR DESCRIPTION
**Pull request by XiShanYongYe-Chang**
Signed-off-by: changzhen <changzhen5@huawei.com>

**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

As https://github.com/karmada-io/karmada/pull/2871 update examples of the `karmadactl interpret` command, so I update it.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/reference/karmadactl/karmadactl-usage-conventions.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 ---
-title: Karmadactl Usage Conventions
+title: Karmadactl使用约定
 ---
 
-Recommended usage conventions for `karmadactl`.
+`karmadactl`使用约定建议说明。
```
Comment (RainbowMango) [inline i18n/zh/docusaurus-plugin-content-docs/current/reference/karmadactl/karmadactl-usage-conventions.md:5]: 机器翻的吗？

Comment (Poor12) [reply i18n/zh/docusaurus-plugin-content-docs/current/reference/karmadactl/karmadactl-usage-conventions.md:5]: K8s用的是推荐用法约定

Comment (XiShanYongYe-Chang) [reply i18n/zh/docusaurus-plugin-content-docs/current/reference/karmadactl/karmadactl-usage-conventions.md:5]: 不是啊，怎么翻好点儿？

**Review by RainbowMango — APPROVED** [review]
/approve

**Pull request merged**